fedora_bot.py

```diff
@@ -81,30 +81,43 @@ def slack_notify(message: str):
     assert response.body == "ok"
 
 
-def merge_open_pull_requests(args, component):
-    req = request.Request(f'https://src.fedoraproject.org/api/0/rpms/{component}/pull-requests?author=packit', method="GET") # returns open PRs created by packit
+def check_pull_request_flags(component, pr_id):
+    """
+    Check the test results in the pull request, which are represented in Pagure as 'flags'
+    As the test results (pagure flags) are not immediately available and there is no indication of running tests
+    we have to hardcode the amount of test results to expect so we can verify all tests have passed.
+    """
+    req = request.Request(f'https://src.fedoraproject.org/api/0/rpms/{component}/pull-request/{pr_id}/flag', method="GET") # returns test statuses
```

Comment:
Might be easuer to use `requests` here as we are already importing it.

Reply:
Yeah, I was actually considering to clean things up and use one library for http requests (I don't have a strong preference for either) in the bot.
Do you have any preferences or suggestions which one to use?

Reply:
`requests`: simpler API, 3rd party but pretty much an industry standard. Nevertheless, can be done in a follow-up.

Reply:
@ondrejbudai I added a second commit with the move from `urllib` to `requests`.